spyder/plugins/editor/confpage.py

```diff
@@ -184,6 +193,33 @@ def setup_page(self):
             'tab_stop_width_spaces',
             default=4, min_=1, max_=8, step=1)
 
+        check_eol_box = newcb(
+            _("Fix mixed EOLs automatically and show warning"),
+            'check_eol_chars',
+            default=True,
+            tip=_(
+                "When opening a file containing mixed end-of-line characters\n"
+                "(which may raise syntax errors in the console on Windows),\n"
+                "Spyder will convert them automatically if enabled."
+            ),
+        )
+        convert_eol_on_save_box = newcb(
+            _("Convert end-of-line characters to the following on save:"),
+            'convert_eol_on_save',
+            default=False,
+        )
+        eol_combo_choices = (
+            (_("LF (Linux/macOS)"), 'LF'),
+            (_("CRLF (Windows)"), 'CRLF'),
```

Comment:
```suggestion
            ("LF (Linux/macOS)", 'LF'),
            ("CRLF (Windows)", 'CRLF'),
```

Nothing to translate here. And I think there's another instance of these strings in our menus, so it'd be nice if you could transform those strings to regular ones too.

Reply:
Okay, that was just how it was before. I went ahead and changed those two as well as the two other instances in `main_widget.py`